Resources/Prototypes/_Goobstation/Entities/Objects/Misc/implanters.yml

```
@@ -84,3 +84,11 @@
   components:
   - type: Implanter
     implant: BluespaceLifelineImplant
+
+- type: entity
+  parent: BaseImplantOnlyImplanter
+  id: PacifismImplanter
+  suffix: pacifism
+  components:
+  - type: Implanter
```

yamllint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)